docs/conf.py

```diff
@@ -1,246 +1,26 @@
 # -*- coding: utf-8 -*-
-#
```

Comment:
For this PR, can we leave the doc conf alone? (or move it to a new PR if changes are necessary?)
I don't _think_ this changes anything substantial, but ... (also I'd leave the comments in personally)

Reply:
@meejah yes, certainly, I will close this PR and prepare a new, content-only PR